api/v1beta1/spannerautoscaleschedule_types.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type Schedule struct {
+	Cron     string `json:"cron"`
+	Duration string `json:"duration"`
+}
+
+// SpannerAutoscaleScheduleSpec defines the desired state of SpannerAutoscaleSchedule
+type SpannerAutoscaleScheduleSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// Foo is an example field of SpannerAutoscaleSchedule. Edit spannerautoscaleschedule_types.go to remove/update
+	Foo string `json:"foo,omitempty"`
```

Comment:
```suggestion
```

I feel that this is not necessary.
Because in https://github.com/mercari/spanner-autoscaler/pull/71/files#diff-e804a70ac7d087584ea5ccb2b5a63a976845e507b00e7dadf8c6c2746a1a93a3R42 ,  the sample field is not set.